app/src/main/java/com/example/kuba10/mypokemonplaces/ListFragment/FirebaseListFragment.java

```diff
@@ -1,151 +0,0 @@
-
-import android.os.Bundle;
-import android.support.v4.app.Fragment;
-import android.support.v7.widget.LinearLayoutManager;
-import android.support.v7.widget.RecyclerView;
-import android.support.v7.widget.helper.ItemTouchHelper;
-import android.view.LayoutInflater;
-import android.view.View;
-import android.view.ViewGroup;
-
-import com.example.kuba10.mypokemonplaces.ListFragmentAdapters.AdapterForTouchAndFirebase;
-import com.example.kuba10.mypokemonplaces.ListFragmentAdapters.CardViewHolder;
-import com.example.kuba10.mypokemonplaces.ListFragmentAdapters.OnStartDragListener;
-import com.example.kuba10.mypokemonplaces.ListFragmentAdapters.SimpleItemTouchHelperCallback;
-import com.example.kuba10.mypokemonplaces.Constants;
-import com.example.kuba10.mypokemonplaces.FragmentListener;
-import com.example.kuba10.mypokemonplaces.Model.PokePlace;
-import com.example.kuba10.mypokemonplaces.Model.PokemonGo;
-import com.example.kuba10.mypokemonplaces.R;
-import com.google.firebase.database.FirebaseDatabase;
-import com.google.firebase.database.Query;
-
-import java.util.ArrayList;
-
-import butterknife.BindView;
-import butterknife.ButterKnife;
-
-
-public class FirebaseListFragment extends Fragment implements OnStartDragListener {
-
-
-    private AdapterForTouchAndFirebase mFirebaseAdapter;
-    private ItemTouchHelper mItemTouchHelper;
-    private FragmentListener fragmentListener;
-    private Query orderByChild;
-
-
-    @BindView(R.id.listRecycler)
-    RecyclerView recyclerView;
-
-
-    public static FirebaseListFragment newInstance() {
-        return new FirebaseListFragment();
-    }
-
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        fragmentListener = (FragmentListener) getActivity();
-
-    }
-
-    @Override
-    public View onCreateView(LayoutInflater inflater, ViewGroup container,
-                             Bundle savedInstanceState) {
-        View view = inflater.inflate(R.layout.fragment_list, container, false);
-        ButterKnife.bind(this, view);
-
-
-        prepareQuery();
-        setUpFirebaseAdapter();
-
-        return view;
-    }
-
-    @Override
-    public void onStart() {
-        super.onStart();
-    }
-
-    private void setUpFirebaseAdapter() {
-
-        mFirebaseAdapter = new AdapterForTouchAndFirebase(PokePlace.class,
-                R.layout.place_card_layout, CardViewHolder.class,
-                orderByChild, this, this);
-
-        recyclerView.setHasFixedSize(true);
-        LinearLayoutManager layoutManager = new LinearLayoutManager(getActivity());
-        recyclerView.setLayoutManager(layoutManager);
-        recyclerView.setAdapter(mFirebaseAdapter);
-
-        ItemTouchHelper.Callback callback = new SimpleItemTouchHelperCallback(mFirebaseAdapter);
-        mItemTouchHelper = new ItemTouchHelper(callback);
-        mItemTouchHelper.attachToRecyclerView(recyclerView);
-    }
-
-    private void prepareQuery() {
```

Comment:
mi osobiście nie podobają się takie metody typu "prepare". Sugerowałbym "generateQuery" albo "getQuery" - metodę która zwróci `Query`, szczególnie, że jest to synchroniczne

Reply:
👍